src/main/java/chess/controller/WebController.java

```diff
@@ -0,0 +1,62 @@
+package chess.controller;
+
+import static chess.view.webview.Converter.convertToWebViewPiece;
+
+import chess.domain.board.position.Position;
+import chess.domain.piece.Piece;
+import chess.dto.ScoreDto;
+import chess.service.ChessGameService;
+import java.util.Map;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class WebController {
+
+    private final ChessGameService chessGameService;
+
+    public WebController(final ChessGameService chessGameService) {
+        this.chessGameService = chessGameService;
+    }
+
+    @GetMapping(path = "/")
+    public String index(final Model model) {
+        final Map<String, Object> pieces = convertToWebViewPiece(chessGameService.getPieces());
+        model.addAllAttributes(pieces);
+        return "index";
+    }
+
+    @GetMapping(path = "/start")
+    public String start(final Model model) {
+        final Map<String, Object> pieces = convertToWebViewPiece(chessGameService.start());
+        model.addAllAttributes(pieces);
+        return "index";
+    }
+
+    @PostMapping(path = "/move")
+    public String move(final Model model,
+                     @RequestParam("source") String source,
+                     @RequestParam("target") String target) {
+        final Map<Position, Piece> pieces = chessGameService.move(source, target);
+        model.addAllAttributes(convertToWebViewPiece(pieces));
+        return "index";
+    }
```

Comment:
현재 Post 메서드 전달 후, 페이지를 반환하고 있는데요.
브라우저에서 뒤로가기를 클릭했을 때, 다시한번 post를 요청할 수 있습니다. 
이걸 해결하기위해 post-redirect-get 이라는 패턴을 사용하는데요. 아래처럼 적용해보시면 좋을 것 같아요.

```suggestion
    @PostMapping(path = "/move")
    public String move(final Model model,
                     @RequestParam("source") String source,
                     @RequestParam("target") String target) {
        chessGameService.move(source, target);
        return "redirect:/";
    }
```

https://en.wikipedia.org/wiki/Post/Redirect/Get